### ISSUE TITLE
Heading structure of sections on Edit screens in CMS

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/va.gov-cms#10112]
## Description
In the editorial experience in the CMS, there is an edit screen for each content type.  Each content type generally contains multiple editable sections but the heading structure of these sections currently is not consistent.  Some sections have headings, some do not.  The sections that do have headings are H4 which skips the H3 level in the overall heading structure for the page.

## Screenshot
![image](https://user-images.githubusercontent.com/106678594/183672835-07b31eaa-e70f-4512-8517-e5062540fd8e.png)

## Accessibility Standard

WCAG version 2.0 AA, [Criterion 1.3.1](https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships.html)

## Acceptance Criteria
- [ ] Technical feasibility discussion to determine what can be changed regarding these section headings, how much flexibility is available within the section itself and between content types
- [ ] Update Design System to created standard heading structure
- [ ] Second Technical feasibility review to confirm desired changes
- [ ] Change management consulted
- [ ] Implementation ticket created

### CMS Team
Please check the team(s) that will do this work.

- [ ] `Program`
- [ ] `Platform CMS Team`
- [ ] `Sitewide Crew`
- [ ] `⭐️ Sitewide CMS`
- [ ] `⭐️ Public Websites`
- [ ] `⭐️ Facilities`
- [ ] `⭐️ User support`

**Comment by srancour** [comment]
This is still an issue and something we should dig into.

**Comment by gracekretschmer-metrostar** [comment]
Impact: creates confusion for screen reader users because they think they missed something. Should be an "easy" fix. Will need to do testing to confirm the scope of bug and impact of fix.

**Comment by gracekretschmer-metrostar** [comment]
Impact: creates friction for end-users but will be a quick win to fix. Good learning experience for drupal developers. 

in the "should category"

**Comment by gracekretschmer-metrostar** [comment]
@anantais will work on this for sprint 10 and collaborate with Steve on how to move forward.
